src/pages/nft/class/_classId/_nftId.vue

```diff
@@ -314,30 +314,17 @@
             @click-user-collected-count="handleClickUserCollectedCount"
           />
         </section>
-
-        <!-- recommend -->
-        <div
-          v-if="isRecommendationLoading"
-          class="flex justify-center items-center my-[24px]"
-        >
-          <ProgressIndicator />
-        </div>
-        <NFTPageRecommendation
-          v-else
-          :iscn-owner="iscnOwner"
-          :iscn-work-author="iscnWorkAuthor"
-          :is-followed="isFollowed"
-          :should-show-follow-button="shouldShowFollowButton"
-          :recommended-list="recommendedList"
-          :is-book-nft="nftIsNFTBook"
-          @header-avatar-click="handleRecommendationHeaderAvatarClick"
-          @follow-button-click="handleFollowButtonClick"
-          @item-click="handleRecommendedItemClick"
-          @item-collect="handleRecommendedItemCollect"
-          @slide-next.once="handleRecommendationSlideNext"
-          @slide-prev.once="handleRecommendationSlidePrev"
-          @slider-move.once="handleRecommendationSliderMove"
-        />
+        <client-only>
```

Comment:
Just curious, what is populated only in client but not in server in this component?

Reply:
I'm not really sure why, but it blocks the entire page

Reply:
If all `NFTPageRecommendation` need to be in `client-only`, can we just put `client-only` inside `NFTPageRecommendation`?
Even better if we can figure out which part of `NFTPageRecommendation` is causing hydration error, and just wrap that part in `client-only`